src/tools/auth0/handlers/attackProtection.js

```diff
@@ -0,0 +1,66 @@
+import DefaultHandler from './default';
+
+export const schema = {
+  type: 'object'
+};
+
+export default class AttackProtectionHandler extends DefaultHandler {
+  constructor(config) {
+    super({
+      ...config,
+      type: 'attackProtection'
+    });
+  }
+
+  objString(item) {
+    return super.objString({
+      'breached-password-protection': {
+        enabled: item.breachedPasswordDetection.enabled
+      },
+      'brute-force-protection': {
+        enabled: item.bruteForceProtection.enabled
+      },
+      'suspicious-ip-throttling': {
+        enabled: item.suspiciousIpThrottling.enabled
+      }
+    });
+  }
+
+  async getType() {
+    if (this.existing) {
+      return this.existing;
+    }
+
+    const breachedPasswordDetection = await this.client.attackProtection.getBreachedPasswordDetectionConfig();
+    const bruteForceProtection = await this.client.attackProtection.getBruteForceConfig();
+    const suspiciousIpThrottling = await this.client.attackProtection.getSuspiciousIpThrottlingConfig();
```

Comment:
Not obvious to a JS newcomer, but these request will execute in serial. To parallelize, I recommend leveraging `Promise.all()` see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Example:

```
const [breachedPasswordDetection, bruteForceProtection, suspiciousIpThrottling] = await Promise.all([
this.client.attackProtection.getBreachedPasswordDetectionConfig(),
this.client.attackProtection.getBruteForceConfig(),
this.client.attackProtection.getSuspiciousIpThrottlingConfig()
])
```

^ Super condensed, would be valid to break out into separate variables and steps.

Reply:
Using promise.all through https://github.com/auth0/auth0-deploy-cli/pull/428/commits/6a8d1db6e55baf79521502938d3d96a08d54ad18